### PR TITLE
Add registered model served by mlflow as example model on port 1234

### PR DESCRIPTION
**Pull request by avan1235**

### CODE REVIEW

**Comment by avan1235** [comment]
Clear you environment before building. It uses now debian slim version for nussknacker so image rebuild is needed

File: dev-environment/docker-compose.yml [hunk 1]
```diff
@@ -4,19 +4,56 @@ services:
 
   mlflow:
     build: ./mlflow
-    container_name: mlflow_server
+    container_name: mlflow-server
     restart: always
     ports:
-     - "5000:5000"
+      - "5000:5000"
```
Comment (kantoniak) [inline dev-environment/docker-compose.yml:10]: Let's add a YAML checker at some point to prevent formatting changes.

**Review by kantoniak — APPROVED** [review]
I would stick to `iris` dataset to try glue MLFlow with Nussknacker sample, but wine is fine too.

File: dev-environment/docker-compose.yml (hunk 1 above)
Comment (avan1235) [reply dev-environment/docker-compose.yml:10]: ok, good point, will remember

**Comment by avan1235** [comment]
Can change to iris in future commits to keep repository clean but leave this sample for know as it works and gives the ability to test the workflow

**Pull request merged**